hydromt_wflow/wflow.py

```diff
@@ -2715,6 +2716,267 @@ def setup_precip_forcing(
             precip_out.attrs.update({"precip_clim_fn": precip_clim_fn})
         self.set_forcing(precip_out.where(mask), name="precip")
 
+    def setup_precip_from_point_timeseries(
+        self,
+        precip_fn: Union[str, pd.Series, pd.DataFrame, xr.Dataset],
```

Comment:
```suggestion
        precip_fn: Union[str, pd.DataFrame, xr.Dataset],
```
The code is complex enough as is so I do not think we need to support pd.Series. I think the get_dataframe method retunrs a dataframe directly and if people want ot pass a python object directly, they can convert it themselves first to a dataframe. If you do not agree let me know!

Reply:
Allright! Maybe I thought that get_dataframe would return a series when only 1 stations was included :)